services/github/schemas/v1/taskcluster-render-yaml-output.yml

```diff
@@ -0,0 +1,22 @@
+$schema: "/schemas/common/metaschema.json#"
+title: "Render .taskcluster.yml output"
+description: Rendered .taskcluster.yml output.
+type: object
+properties:
+  tasks:
+    type: array
+    description: Rendered task
+    uniqueItems: false
+    items:
+      type: object
```

Comment:
why not a task definition?

Reply:
go codegen didn't like me initially, will see if I can make him love me again

Reply:
I decided against mixing queue entities into github service to be consistent with design and existing functionality. Github currently doesn't validate produced tasks that it later sends to the queue service to create such tasks. So I think it's better to keep it this way, and if someone wants to check the generated payload, he is able to do it once he gets rendered tasks out.
Btw, this is now implemented in the UI section. UI uses github service to render yaml into tasks, and then validates those tasks against `create-task-request.json` schema, and presents them in the UI